src/SFA.DAS.Reservations.Web/Controllers/ReservationsController.cs

```diff
@@ -278,12 +282,25 @@ public IActionResult PostCompleted(ReservationsRouteModel routeModel, Confirmati
                 return View(viewModel.ViewName, viewModel);
             }
 
-            if (model.AddApprentice.HasValue && model.AddApprentice.Value)
+            if (!string.IsNullOrEmpty(model.WhatsNext) && !string.IsNullOrWhiteSpace(model.WhatsNext))
             {
-                return Redirect(model.ApprenticeUrl);
+                
+                switch (model.WhatsNext)
+                {
+                    case ConfirmationRedirectViewModel.RedirectOptions.RecruitAnApprentice:
+                        var url = _urlHelper.GenerateUrl(subDomain: "recruit",id: routeModel.UkPrn.ToString());
```

Comment:
I would look at moving this into the model creation of the GET before hand like the other ones are doing

Reply:
I've now implemented this change